miscelanea/005-deploy-in-one-pod/freeipa-admin.yaml.envsubst

```diff
@@ -0,0 +1,161 @@
+---
+# https://docs.openshift.com/container-platform/4.5/authentication/managing-security-context-constraints.html
+# https://kubernetes-security.info/
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: freeipa
+  labels:
+    app: freeipa
+  annotations:
+    kubernetes.io/description: Provides a Freeipa Pod deployed all in one
+      just for investigation prupose.
+    release.openshift.io/create-only: "true"
+allowHostDirVolumePlugin: true
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: false
+allowedCapabilities:
+  # Default capabilities anyuid
+  - "SETUID"
+  - "SETGID"
+  - "FSETID"
+  - "SETPCAP"
+  - "DAC_OVERRIDE"
+  - "NET_RAW"
+  - "NET_BIND_SERVICE"
+  - "SYS_CHROOT"
+  - "KILL"
+  - "AUDIT_WRITE"
+  - "CHOWN"
+  - "FOWNER"
+  - "SETFCAP"
+
+  # No default capabilities
+  - "SYS_ADMIN"
+  - "SYS_RESOURCE"
+  - "MKNOD"
+allowedUnsafeSysctls:
+defaultAddCapabilities:
+  # Default capabilities anyuid
+  - "CHOWN"
+  - "FOWNER"
+  - "SETFCAP"
+
+  - "SETPCAP"
+  - "SETFCAP"
+  - "SETUID"
+  - "SETGID"
+  - "DAC_OVERRIDE"
+  - "NET_BIND_SERVICE"
+  - "KILL"
+
+  # No default capabilities
+  - "SYS_ADMIN"
+  - "SYS_RESOURCE"
+  - "MKNOD"
+fsGroup:
+  type: RunAsAny
+priority: 20
+readOnlyRootFilesystem: false
+requiredDropCapabilities: null
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: MustRunAs
+supplementalGroups:
+  type: RunAsAny
+users:
+  # - system:serviceaccount:${NAMESPACE}:feeipa
+  - freeipa
+volumes:
+  - configMap
+  - downwardAPI
+  - emptyDir
+  - persistentVolumeClaim
+  - projected
+  - secret
+  - hostPath
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: freeipa
+  labels:
+    app: freeipa
+automountServiceAccountToken: true
```

Comment:
This is used to talk to kubernetes API in initContainer.  BUT checkov complains about it.  So we should investigate if there is a better way.

Reply:
Internal ticket: https://issues.redhat.com/browse/IDMOCP-174 . Resolving comment.